lib/webmention/client.rb

```diff
@@ -63,25 +59,21 @@ def send_mentions
     # target - The link that was mentioned in the source page.
     #
     # Returns a boolean.
-    def self.send_mention endpoint, source, target, full_response=false
+    def self.send_mention(endpoint, source, target, full_response = false)
       data = {
-        :source => source,
-        :target => target,
+        source: source,
+        target: target
       }
 
       # Ensure the endpoint is an absolute URL
-      endpoint = absolute_endpoint endpoint, target
+      endpoint = absolute_endpoint(endpoint, target)
 
       begin
-        response = HTTParty.post(endpoint, {
-          :body => data
-        })
```

Comment:
Ruby plays pretty fast and loose with hash parameters. In this case, `HTTParty.post(endpoint, body: data)` is equivalent.